keras_nlp/src/models/resnet_vd/resnet_vd_backbone_tests.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import keras
+import numpy as np
+import pytest
+from absl.testing import parameterized
+
+from keras_nlp.src.models.resnet_vd.resnet_vd_backbone import ResNetVdBackbone
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class ResNetVdBackboneTest(TestCase):
+    def setUp(self):
+        self.input_batch = np.ones(shape=(2, 224, 224, 3))
+
+    def test_valid_call(self):
+        model = ResNetVdBackbone(
+            stackwise_filters=[64, 128, 256, 512],
+            stackwise_blocks=[2, 2, 2, 2],
+            stackwise_strides=[1, 2, 2, 2],
+            include_rescaling=False,
+        )
+        model(self.input_batch)
+
+    def test_valid_call_with_rescaling(self):
+        model = ResNetVdBackbone(
+            stackwise_filters=[64, 128, 256, 512],
+            stackwise_blocks=[2, 2, 2, 2],
+            stackwise_strides=[1, 2, 2, 2],
+            include_rescaling=True,
+        )
+        model(self.input_batch)
+
+    @pytest.mark.large  # Saving is slow, so mark these large.
+    def test_saved_model(self):
```

Comment:
Use KerasNLPs standard test routine like this for the saved model test - https://github.com/keras-team/keras-nlp/blob/keras-hub/keras_nlp/src/models/vgg/vgg_backbone_test.py#L44